internal/servers/controller/handlers/mask_manager.go

```diff
@@ -30,26 +32,28 @@ func NewMaskManager(dest protoreflect.ProtoMessage, src ...protoreflect.ProtoMes
 	for k, v := range srcToDest {
 		ov, ok := destToSrc[v]
 		if !ok || ov != k {
-			return nil, fmt.Errorf("mapping src field %q maps to %q, dest %q maps to %q", k, v, v, ov)
+			return nil, errors.New(errors.Encrypt, op, fmt.Sprintf("mapping src field %q maps to %q, dest %q maps to %q", k, v, v, ov))
 		}
 		result[k] = v
 	}
 
 	// Now check to make sure there aren't any dangling dest mappings.
 	for k, v := range destToSrc {
 		if ov, ok := srcToDest[v]; !ok || ov != k {
-			return nil, fmt.Errorf("mapping dest field %q maps to %q, src %q maps to %q", k, v, v, ov)
+			//todo(schristoff): I'm not in love with this error
+			return nil, errors.New(errors.Encrypt, op, fmt.Sprintf("mapping src field %q maps to %q, dest %q maps to %q", k, v, v, ov))
```

Comment:
So I agree, but not sure I have a great alternative.  I have had to read this a couple of times to figure it out.  

Also I am not a fan of the `ov` naming here as it creates confusion around `ok` - `'ok` here is the map lookup bool and meant to be`okay` not `o-k` like `o-v` is trying to be.  My first read of this was that `ok` was the second key like `ov` is the second value...

Reply:
yes I had much trouble reading this as well